The build was failing in #2814.

https://github.com/groonga/groonga/actions/runs/26208494874/job/77113528887?pr=2814#step:9:19453

```
[1](https://github.com/groonga/groonga/actions/runs/26208494874/job/77113528887?pr=2814#step:9:19452)
make[8]: Entering directory '/home/runner/work/groonga/groonga/doc/locale/en'
cp: cannot stat './html/_static/vendor/fontawesome/webfonts/fa-brands-400.ttf': No such file or directory
make[8]: *** [Makefile:5813: distdir-am] Error 1
```

This fixes it.

--- a/doc/files.am
+++ b/doc/files.am
@@ -4723,11 +4723,9 @@ html_files_relative_from_locale_dir = \
 	html/_images/used-when-indexing.png \
 	html/_images/used-when-searching.png \
 	html/_images/zulip-icon-128x128.png \
-	html/_static/base-stemmer.js \
 	html/_static/basic.css \
 	html/_static/doctools.js \
 	html/_static/documentation_options.js \
-	html/_static/english-stemmer.js \
 	html/_static/favicon.ico \
 	html/_static/file.png \
 	html/_static/groonga.png \

--- a/doc/files.cmake
+++ b/doc/files.cmake
@@ -1393,11 +1393,9 @@ set(GRN_DOC_HTML_FILES
     _images/used-when-indexing.png
     _images/used-when-searching.png
     _images/zulip-icon-128x128.png
-    _static/base-stemmer.js
     _static/basic.css
     _static/doctools.js
     _static/documentation_options.js
-    _static/english-stemmer.js
     _static/favicon.ico
     _static/file.png
     _static/groonga.png

